charts/dex.yaml

```diff
@@ -0,0 +1,56 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/infra: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: '/$2'
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    kubernetes.io/ingress.class: "nginx-control-plane"
+  path: /oidc(/|$)(.*)
+  hosts:
+    - null
+  tls:
+   - secretName: dex-tls-secret
+
+certs:
+  web:
+    create: false
+  grpc:
+    create: false
+
+config:
+  issuer: https://{{ pillar['metalk8s']['api_server']['host'] }}:32000
+
+  web:
+    # port is taken from ports section above
+    address: 0.0.0.0
+    # tlsCert: /etc/dex/tls/https/server/tls.crt
+    # tlsKey: /etc/dex/tls/https/server/tls.key
+  connectors: {}
+
+  staticPasswords:
+    - email: "admin@example.com"
```

Comment:
I commented on this before but the comment seems to be gone?

Reply:
I was not marked resolved yet

Reply:
Sure, but you did make a change to the line, and as such GitHub removed the comment from the diff view. Which can be considered an issue in the GitHub UI.

Reply:
What's this e-mail used for anyway? Could it be `admin@metalk8s.invalid` or something along those lines?

Reply:
The e-mail address is used in the OIDC profile of this 'user', and as such can show up e.g. in Grafana.

It should indeed be set to some 'invalid' address.